packages/slice-machine/src/features/customTypes/EditDropdown.tsx

```diff
@@ -75,7 +75,7 @@ export const EditDropdown: FC<EditDropdownProps> = ({
           <Button
             loading={isCustomTypeBeingConverted}
             startIcon="moreVert"
-            variant="secondary"
+            color="grey"
             data-testid="editDropdown"
           />
```

Comment:
Should this be an IconButton instead? If so:

Suggested label: "Actions" or "Custom type actions"

(I'm not sure if this component is reused with page types. If so, we need to label it correctly.)

Reply:
Yes you're right, IconButton works. I guess it was before the refactoring that allowed variant="solid".

Done for both